backend/src/game/player.py

```diff
@@ -19,10 +19,10 @@ def change_health(self, change_integer):
         self.current_health += change_integer
         if self.current_health > self.maximum_health:
             self.current_health = self.maximum_health
-        if self.current_health <= constants.ZERO:
+        if self.current_health <= constants.DEATH_HIT_POINT:
             self.is_alive = False
 
     def change_victory_points(self, change_integer):
```

Comment:
This is outside the scope of this PR but these functions "change_*" would imply that you are taking the value and changing it to the value that is given.  eg. If you victory_points = 10 and you see change_victory_points(3), my mind would think it is changing the value to 3 not adding to it to make it 13.  Would need to change up the language to make this much clearer

Reply:
Ok I can see that, no one brought it up in the Player PR.
I'll see what I can do, but if you have a better name suggestion, please write.

Reply:
The best I can think of off the top of my head would just be 'change_value_by' or 'update_value_by', would keep the same functionality under the covers but the implication is your changing it BY that value not TO it

Reply:
I like the `update_value_by` approach, makes it clear to handle increments/decrements